instructure/canvas-android
# [Flutter-Parent][MBL-14368] Fix webview crash when really large

* No longer embedding web views with other native content. Adds a tile to view description in another screen. Added a simple new screen to show html (rather than a url)
* Adds refresh icon to toolbar in course details (when syllabus or home page are visible) to refresh the course data (since PTR doesn't work on full screen web views).

To test:
See ticket for original crash details.
Also test assignment empty states (shows the "There's no description yet" message)
Event empty states (should not show any message with an empty description)
Announcements don't have empty states

Announcements, assignments, and events should all have a button to view web content, rather than an embedded webview.

Course details for Front Page and Syllabus should show a full screen webview (like before) but no pull to refresh functionality right now. Instead, the refresh button added to the course details should reload those screens with new content.
